.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: PR builder
+
+on:
+  pull_request:
+    branches:
+    - develop
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: styfle/cancel-workflow-action@0.6.0
+      with:
+        access_token: ${{ github.token }}
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.19'
+    - name: cache node modules
+      id: cache
+      uses: actions/cache@v2
+      with:
+        path: node_modules
+        key: node-modules-${{ hashFiles('package-lock.json') }}
```

Comment:
Would yarn instructions generate this file?

Reply:
good catch!